go.mod

```diff
@@ -7,8 +7,7 @@ replace github.com/spf13/viper => github.com/istio/viper v1.3.3-0.20190515210538
 // Old version had no license
 replace github.com/chzyer/logex => github.com/chzyer/logex v1.1.11-0.20170329064859-445be9e134b2
 
-// Avoid pulling in incompatible libraries
-replace github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
+replace github.com/docker/distribution => github.com/docker/distribution v2.7.1+incompatible
```

Comment:
I have some concerns about blindly replacing with a new library, and how it may impact dependencies that user other versions. It seems ideally we would get the downstream projects to update? Seems like we mostly need https://github.com/helm/helm/pull/9675

Reply:
Opened https://github.com/kubernetes/kubernetes/issues/103849 as well.

In particular, I tried changing helm to use `v2.7.1+incompatible`, and it no longer passes their tests. This makes me concerned we may be breaking some functionality depended on

Reply:
fine, I close this.